docs/advanced/advanced.md

```diff
@@ -314,6 +314,40 @@ After the build, view the html with:
 
 `start docs\_build\html\index.html`
 
+(sphinx:manual-assets)=
+## Manually specify extra files/folders to be included in a website
+
+Jupyter Book will copy over any files that are linked from within its pages so that the links work in the built website. However, sometimes you'd like to manually ensure that files and folders are included in your built website. For example, if you'd like to link to them from _outside_ your built documentation, but not from within your built documentation.
```

Comment:
```suggestion
Jupyter Book will copy over any files that are linked from within its pages so that the links work in the built website.
However, sometimes you'd like to manually ensure that files and folders are included in your built website.
For example, if you'd like to link to them from *outside* your built documentation, but not from within your built documentation.
```

one sentence per line 😉

Reply:
I thought we were shooting for one paragraph per line in order to make it easier to make edits etc 🤷